docs/getting-started.md

```diff
@@ -21,6 +21,8 @@ Then build the manifests and images:
 make && make push && make manifests
 ```
 
+**Note:** If the `make && make push && make manifests` command gives an unexpected disconnect or Early EOF while fetching the repositories during the building of manifests and images, then adjusting the default timeout using `export PULLER_TIMEOUT=10000` will solve the issue. 
```

Comment:
I'd follow the line length of the `Getting Started` section.

I'd rephrase this as something like:

````markdown
**Note:** If you see failures related to fetching some modules, try increasing bazel's timeout with:
 ```bash
 export PULLER_TIMEOUT=10000
 ```
````

Reply:
I have rephrased the line according to your suggestion